examples/passage_ranker/bert_ranker.py

```diff
@@ -19,10 +19,10 @@
 from typing import Optional, cast
 
 import torch
-from torch.nn import Parameter
-from texar.torch.modules.pretrained import PretrainedBERTMixin
-from texar.torch.modules.encoders import BERTEncoder as TxBERTEncoder
 from texar.torch.modules.classifiers import BERTClassifier as TxBERTClassifier
+from texar.torch.modules.encoders import BERTEncoder as TxBERTEncoder
+from texar.torch.modules.pretrained import PretrainedBERTMixin
+from torch.nn import Parameter
```

Comment:
I think you have to put `from torch.nn import Parameter` together with `import torch` to align with the grouping rule for python imports. In fact, you don't have to modify the files that you haven't touched right? They should have passed pylint and flake8 already.

Reply:
Restored.